src/Payments.Core/PaymentsContext.cs

```diff
@@ -8,8 +9,14 @@ public class PaymentsContext : DbContext
         public PaymentsContext(DbContextOptions<PaymentsContext> options) : base(options)
         {
             
+        }
+
+        [Obsolete("Just use for tests")]
+        public PaymentsContext()
+        {
+
         }
 
-        public DbSet<Invoice> Invoices { get; set; }
+        public virtual DbSet<Invoice> Invoices { get; set; }
```

Comment:
@srkirkland 

Test Name:	Payments.Tests.ControllerTests.InvoiceControllerTests.Test1
Test FullName:	Payments.Tests.ControllerTests.InvoiceControllerTests.Test1
Test Source:	C:\GitProjects\payments\tests\Payments.Tests\ControllerTests\InvoiceControllerTests.cs : line 48
Test Outcome:	Failed
Test Duration:	0:00:00.119

Result StackTrace:	
at Moq.Mock.ThrowIfCantOverride(Expression setup, MethodInfo method)
   at Moq.Mock.<>c__DisplayClass66_0`2.<SetupGet>b__0()
   at Moq.PexProtector.Invoke[T](Func`1 function)
   at Moq.Mock.SetupGet[T,TProperty](Mock`1 mock, Expression`1 expression, Condition condition)
   at Moq.Mock.<>c__DisplayClass65_0`2.<Setup>b__0()
   at Moq.PexProtector.Invoke[T](Func`1 function)
   at Moq.Mock.Setup[T,TResult](Mock`1 mock, Expression`1 expression, Condition condition)
   at Moq.Mock`1.Setup[TResult](Expression`1 expression)
   at Payments.Tests.ControllerTests.InvoiceControllerTests.Test1() in C:\GitProjects\payments\tests\Payments.Tests\ControllerTests\InvoiceControllerTests.cs:line 61
Result Message:	Invalid setup on a non-virtual (overridable in VB) member: m => m.Invoices